shared/message/output/CompressedImage.proto

```diff
@@ -33,20 +33,28 @@ message CompressedImage {
             EQUIDISTANT = 2;
             EQUISOLID   = 3;
         }
-
-        Projection projection   = 1;
-        float      focal_length = 2;
-        float      fov          = 3;
-        fvec2      centre       = 4;
-        fvec2      k            = 5;
+        Projection projection = 1;
+        /// Normalised focal length: focal length in pixels / image width
+        float focal_length = 2;
+        /// The angular diameter that the lens covers (the area that light hits on the sensor). In radians
+        float fov = 3;
+        /// Normalised image centre offset: pixels from centre to optical axis / image width
+        fvec2 centre = 4;
+        fvec2 k      = 5;
     }
-
-    uint32                    format     = 1;
-    uvec2                     dimensions = 2;
-    bytes                     data       = 3;
-    uint32                    id         = 4;
-    string                    name       = 5;
-    google.protobuf.Timestamp timestamp  = 6;
-    mat4                      Hcw        = 7;
-    Lens                      lens       = 8;
+    /// Encoding of colour channels, e.g. BGGR, PRG8, RGBA, etc.
+    /// see module::output::compressor::turbojpeg::Compressor::compress for full list we use
+    uint32 format = 1;
+    /// Number of pixels of {x, y} of the image
+    uvec2  dimensions = 2;
+    bytes  data       = 3;
+    uint32 id         = 4;
+    /// Name of the camera that took the image
+    string name = 5;
+    /// Time when the image was taken
+    google.protobuf.Timestamp timestamp = 6;
+    /// World to camera Affine3D transform
+    mat4 Hcw = 7;
+    /// The lens information of the camera which took the image
```

Comment:
```suggestion
    /// Lens information of the camera that took the image
```

Reply:
For these it's about deciding if we're using articles (_the_ lens..., _an_ object..., etc) or not (lens..., object..., etc). Went with the latter but it doesn't matter as long as we're consistent here and in other places.